Simplified/Reader2/ReaderStackConfiguration/LibraryService.swift

```diff
@@ -24,14 +61,20 @@ import R2Streamer
 final class LibraryService: Loggable {
   
   private let streamer: Streamer
-  private let publicationServer: PublicationServer
+  let publicationServer: PublicationServing
   private var drmLibraryServices = [DRMLibraryService]()
   
   private lazy var documentDirectory = try! FileManager.default.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: true)
   
-  init(publicationServer: PublicationServer) {
-    self.publicationServer = publicationServer
-    
+  init() {
+    guard let server = (PublicationServerFactory() as PublicationServerMaking).make() else {
```

Comment:
this is the opposite of dependency injection, so it's a "bad practice" per se. I only did it to control the creation of the PublicationServer where it's actually needed. This also allows us to keep those workaround protocols `private`. This code is hopefully temporary anyway (last famous words).

I can also move it out where it was if we want.